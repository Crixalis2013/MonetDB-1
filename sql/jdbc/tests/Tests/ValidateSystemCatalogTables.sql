\vsci
\vsi sys
\vsi tmp
\vdbi

