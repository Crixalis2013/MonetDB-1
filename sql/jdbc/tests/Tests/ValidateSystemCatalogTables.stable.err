stderr of test 'ValidateSystemCatalogTables` in directory 'sql/jdbc/tests` itself:


# 16:25:51 >  
# 16:25:51 >  "./ValidateSystemCatalogTables.SQL.sh" "ValidateSystemCatalogTables"
# 16:25:51 >  


# 16:25:51 >  
# 16:25:51 >  java nl.cwi.monetdb.client.JdbcClient -h localhost -p 34154 -d mTests_sql_jdbc_tests -e -f /home/dinther/dev/dev/MonetDB/sql/jdbc/tests/Tests/ValidateSystemCatalogTables.sql
# 16:25:51 >  


# 16:25:55 >  
# 16:25:55 >  "Done."
# 16:25:55 >  

