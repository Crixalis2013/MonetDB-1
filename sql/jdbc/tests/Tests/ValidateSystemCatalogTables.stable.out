stdout of test 'ValidateSystemCatalogTables` in directory 'sql/jdbc/tests` itself:


# 16:25:09 >  
# 16:25:09 >  "./ValidateSystemCatalogTables.SQL.sh" "ValidateSystemCatalogTables"
# 16:25:09 >  


# 16:25:09 >  
# 16:25:09 >  java nl.cwi.monetdb.client.JdbcClient -h localhost -p 34154 -d mTests_sql_jdbc_tests -e -f /home/dinther/dev/dev/MonetDB/sql/jdbc/tests/Tests/ValidateSystemCatalogTables.sql
# 16:25:09 >  

\vsci
Checking     45 tables/keys  in schema sys for Primary Key uniqueness violations.
Checking     45 columns      in schema sys for Primary Key Not Null violations.
Checking     30 tables/keys  in schema sys for Unique Constraint violations.
Checking     95 foreign keys in schema sys for Foreign Key referential integrity violations.
Checking    169 columns      in schema sys for Not Null violations.
Checking    153 columns      in schema sys for Max Character Length violations.
Checking      6 tables/keys  in schema tmp for Primary Key uniqueness violations.
Checking      6 columns      in schema tmp for Primary Key Not Null violations.
Checking      7 tables/keys  in schema tmp for Unique Constraint violations.
Checking     13 foreign keys in schema tmp for Foreign Key referential integrity violations.
Checking     35 columns      in schema tmp for Not Null violations.
Checking     14 columns      in schema tmp for Max Character Length violations.
\vsi sys
Checking     10 keys         in schema sys for Primary Key uniqueness violations.
Checking      7 keys         in schema sys for Unique Constraint violations.
Checking      0 foreign keys in schema sys for Foreign Key referential integrity violations.
Checking     29 columns      in schema sys for Not Null violations.
Checking    153 columns      in schema sys for Max Character Length violations.
\vsi tmp
Checking      0 keys         in schema tmp for Primary Key uniqueness violations.
Checking      0 keys         in schema tmp for Unique Constraint violations.
Checking      0 foreign keys in schema tmp for Foreign Key referential integrity violations.
Checking      0 columns      in schema tmp for Not Null violations.
Checking     14 columns      in schema tmp for Max Character Length violations.
\vdbi
No user schemas found in this database.


# 16:25:12 >  
# 16:25:12 >  "Done."
# 16:25:12 >  

